pkg/cmd/roachtest/test_runner.go

```diff
@@ -1171,8 +1171,8 @@ func PredecessorVersion(buildVersion version.Version) (string, error) {
 	// (see runVersionUpgrade). The same is true for adding a new key to this
 	// map.
 	verMap := map[string]string{
-		"20.2": "20.1.6",
-		"20.1": "19.2.9",
+		"20.2": "20.1.7",
+		"20.1": "19.2.11",
```

Comment:
I only see the fixtures in `pkg/cmd/roachtest/fixtures/*/checkpoint-v20.1.tgz` being updated, but it seems we're also bumping the version map for 19.2.11? If I've understood correctly, we'll need to generate fixtures for that too. We can do so by changing `newV` below for one run and copying those artifacts in.

Reply:
Ah, thank you! Done :)